pkg/storage/azureblob/getter.go

```diff
@@ -0,0 +1,94 @@
+package azureblob
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/observ"
+)
+
+// Info implements the (./pkg/storage).Getter interface
+func (s *Storage) Info(ctx context.Context, module string, version string) ([]byte, error) {
+	const op errors.Op = "azureblob.Info"
+	ctx, span := observ.StartSpan(ctx, op.String())
+	defer span.End()
+
+	exists, err := s.Exists(ctx, module, version)
+	if err != nil || !exists {
```

Comment:
Could you split this two error handling  to look like in other method (`GoMod` and `Zip`)?